app/models/order_detail.rb

```diff
@@ -461,11 +461,11 @@ def force_complete!
     update(state: "complete", order_status: OrderStatus.complete_status)
   end
 
-  def backdate_to_complete!(event_time = Time.zone.now)
+  def backdate_to_complete!(event_time = Time.zone.now, set_actuals = true)
```

Comment:
How do you feel about changing the interface to this method so it could be called as `@order_detail.backdate_to_complete!(@reservation.reserve_end_at, set_actuals: false)` below, to make it a bit more self-documenting?